src/twisted/web/topfiles/8992.bugfix

```diff
@@ -0,0 +1 @@
+twisted.web's HTTP server no longer leaks file descriptors when timing out inactive clients using TLS connections.
```

Comment:
Style guide says the subject should be a FQPN.  So perhaps "twisted.web.http.HTTPChannel now ...".  Also, given the comment about how the code is still using `loseConnection` and not `abortConnection`, maybe it's better to claim it tries to fix one case and not imply that connections never leak at all with the new version.